site/topics/logic/truth-tables.rst

```diff
@@ -4,9 +4,33 @@ Truth Tables
 
 
 
+Boolean Logic
+=============
+
+
+Boolean Algebra
+---------------
+
+
+
+Truth Tables
+============
+
+
+
+Properties of Logical Operators
+===============================
+
+
+De Morgan's Law
+---------------
+
+
+
 For Next Time
 =============
 
+* `Watch Ben Eater's video on how transistors work <https://www.youtube.com/watch?v=DXvAlwMAxiA>`_
```

Comment:
#16  bad merge here I guess?

Reply:
Nah, I added that too. This one is transistors, the one for the previous topic is semiconductors.